src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import './TodoInfo.scss';
+
+import { Todo } from '../../types/Todo';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
```

Comment:
Consider renaming `TodoInfo` to a more descriptive name that reflects its purpose. 🤔